packages/@okta/vuepress-site/docs/guides/build-sso-integration/before-you-begin/openidconnect/prep.md

```diff
@@ -1,24 +1,22 @@
 
 This multi-tenant approach is different from other IdPs that use a global credential system, where a given application has the same customer credentials across all orgs.
 
-Before you create a new OIDC integration in Okta:
-
-<!-- [ian 2020.02.25] this step doesn't matter unless the user can choose something besides the web as a platform
-1. Decide which platform you'll be using for the integration:
-   * A web application is accessed through the browser and can remain running on a server that can store a secret safely.
-   * A native application resides on the end user's device.
-   * A single-page app (SPA) is a web application that is contained on a single web page. All code is retrieved when the page is loaded initially - the page doesn't reload or refresh. A SPA application cannot keep running on a server.
--->
+Now, before you create a new OIDC integration in Okta:
 
 1. Have your application developed and tested, with a front-end (for example, JavaScript and HTML) and back-end (for example, middleware and database software) stack, along with services available through APIs, and accepting HTTP connections.
 1. Based on the [type of application that you have built](/docs/concepts/oauth-openid/#what-kind-of-client-are-you-building), determine the correct [OAuth 2.0 flow](/docs/concepts/oauth-openid/#recommended-flow-by-application-type) that is required below the OIDC identity layer.
-    >**Note:** For OIDC app integrations intended for publication in the OIN, you must use a Web (server-side) platform with an authorization code flow.
-1. Determine the login redirect URIs on your system. A redirect URI is where Okta sends the authentication response and ID token during the sign-in flow. You can specify more than one URI if required.
+
+    For OIDC applications destined for the OIN, you can create either of the following:
+
+    * A Web application with a dedicated server-side back-end capable of securely storing a Client Secret and exchanging information with an authorization server through trusted back-channel connections. Okta recommends using the Authorization Code flow with an exchange of the client credentials (client ID and Client Secret) for controlling the access between your application and the resource server.
+    * A Single Page Application (SPA), using an Authorization Code flow with a Proof Key for Code Exchange (PKCE) as the recommended method for controlling the access between your application and a resource server.
+
+1. Determine the sign-in redirect URIs on your system. A redirect URI is where Okta sends the authentication response and ID token during the sign-in flow. You can specify more than one URI if required.
```

Comment:
Incorrect Numbering. The line below is also 1.

Reply:
That's normal markdown - if you look at the output https://github.com/okta/okta-developer-docs/pull/2126/files?short_path=a292798#diff-a292798b6571630f448c8ca2a8504af6ee155663049b6290a3aa34a78b30bbfd - the numbering of the steps is correct. (Markdown processes all lines that start with the number 1 in a sequential count).

No change required.